### PR TITLE
Add `circtools-1.2.1`

### PR DESCRIPTION
**Pull request by BarryDigby**
Add a suite of circRNA tools to bioconda. Pypi source: [https://pypi.org/project/circtools/](https://pypi.org/project/circtools/)

Works fine hosted on my own repo/channel/organization(?) [https://anaconda.org/b.digby/circtools](https://anaconda.org/b.digby/circtools)

includes:
- DCC
- FUCHS
- CircTest

### CODE REVIEW

**Comment by BarryDigby** [comment]
@BiocondaBot please fetch artifacts

File: recipes/circtools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - pybedtools >=0.7.10
+    - pysam >=0.13
+    - python
+    - reportlab >=3.3.0
+    - scipy >=0.19.0
+    - statsmodels >=0.9.0
+
+test:
+  commands:
+    - DCC -h
```
Comment (pinin4fjords) [inline recipes/circtools/meta.yaml:36]: Maybe add tests for the other two commands you mentioned?

**Comment by BarryDigby** [comment]
@pinin4fjords re. the `involucro` recipe, I had naively thought this was some sort of artifact from local builds. Should I restore it?

**Comment by pinin4fjords** [comment]
> @pinin4fjords re. the `involucro` recipe, I had naively thought this was some sort of artifact from local builds. Should I restore it?

That or re-do the commits without removing those files and force push, but anything that gets you to a clean PR will be okay.

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@BiocondaBot please merge

**Pull request merged**